management/server/sqlite_store.go

```diff
@@ -134,79 +135,159 @@ func (s *SqliteStore) AcquireAccountLock(accountID string) (unlock func()) {
 	return unlock
 }
 
+func batchInsert(records interface{}, batchSize int, tx *gorm.DB) error {
```

Comment:
Why do we need this wrapper? Doesn't `tx.CreateInBatches` not already do what we want?
If we still need the wrapper, this should ideally be a generic function for compile time type safety.

Reply:
How would you do it?
I don’t want to repeat the code for all objects